syntax = "proto3";

package response_models;

message PlayerIdResponseModel {
	int32 player_id = 1;
}